x-pack/plugins/apm/public/components/shared/Links/apm/TransactionLink.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { pickKeys } from '../../../../../common/utils/pick_keys';
+import { APMLink, APMLinkExtendProps } from './APMLink';
+import { useUrlParams } from '../../../../context/url_params_context/use_url_params';
+import { APMQueryParams } from '../url_helpers';
+
+interface Props extends APMLinkExtendProps {
+  serviceName: string;
+}
+
+const persistedFilters: Array<keyof APMQueryParams> = [
+  'latencyAggregationType',
+];
+
+export function TransactionLink({ serviceName, ...rest }: Props) {
```

Comment:
nit

```suggestion
export function TransactionOverviewLink({ serviceName, ...rest }: Props) {
```

Reply:
@sqren we already have a `TransactionOverviewLink` (https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/shared/Links/apm/TransactionOverviewLink.tsx#L29:L29) This goes to the `Overview` or to the `Transactions`, the one I created only goes to the `Transactions` page.

Reply:
That one (https://github.com/elastic/kibana/blob/master/x-pack/plugins/apm/public/components/shared/Links/apm/TransactionOverviewLink.tsx#L29:L29) should be called `ServiceOrTransactionsOverviewLink` since, like you say, it can go to either of them.